extensions/amp-list/0.1/amp-list.js

```diff
@@ -171,8 +185,22 @@ export class AmpList extends AMP.BaseElement {
     // is missing attributes in the constructor.
     this.initialSrc_ = this.element.getAttribute('src');
 
-    this.container_ = this.createContainer_();
-    this.element.appendChild(this.container_);
+    if (this.element.hasAttribute('diffable')) {
```

Comment:
Dynamic amp-list content can't be SSR'ed. The "initial content" is another way of specifying the placeholder content, except:

1. It doesn't have the standard AMP `[placeholder]` CSS which includes `position: absolute`, so its position can be offset by `[fetch-error]` UI.
2. The initial amp-list fetch and render can be DOM diff'ed against it.

Reply:
I have a few concerns with `initialContent` as I understand. If the initial content is intended to be replaced with the contents from an XHR *without user action*, then it is really a placeholder, and not content. I think it would be beneficial to be able to programmatically differentiate this from the case of inlined content, which will *not* be changed without user interaction, because we can treat them differently from a resizing perspective.

Reply:
As discussed, we'll keep this behavior attribute-gated under `diffable` for `<amp-list>` and consider implementing this as a placeholder feature in future iterations of similar components.